.github/workflows/cd-deploy-nodes-gcp.yml

```diff
@@ -230,7 +230,6 @@ jobs:
         id: auth
         uses: google-github-actions/auth@v2.1.4
         with:
-          retries: '3'
```

Comment:
Are these the deprecated arguments we are removing? Otherwise I'd like to understand why we are we removing the retries

Reply:
Yes, those have been deprecated 
<img width="952" alt="image" src="https://github.com/user-attachments/assets/85dd3b3f-4b3b-4648-ae9b-a9edc8b4f9d3">

Reply:
I've update the PR comment with this info.